### ISSUE TITLE
Remember last read position after closing and reopening the pdf viewer.

### ISSUE THREAD

**Issue by warfarincoumadin** [burtonator/polar-bookshelf#543]
NOTE: Polar is supported by community donations.  Bug reports and features are prioritized for people who make a donation: https://opencollective.com/polar-bookshelf

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by kurmanka** [comment]
I agree with the sentiment, but it is clear that the task is more nuanced than what the issue title says. The idea of pagemarks is very useful, especially for multi-column articles. 

A simpler approach of tracking the furthest read position could be very good too, I believe; but one would probably want to add ability to manually set and reset the furthest read position then (through context menu, for example). 

Simple remembering and restoring the last read position would be good enough in many cases (a basic reading scenario), but one has to decide how it interacts with pagemarks and with progress tracking, if it does.
